src/Mainnav/sgds-mainnav.scss

```diff
@@ -21,4 +23,11 @@ slot[name="non-collapsible"] {
   display: flex;
   gap: 1rem;
   align-items: center;
-}
\ No newline at end of file
+}
+
+.slot-end {
+  display: flex;
+  margin-left: auto;
+  align-items: center;
```

Comment:
the slot end item height is different from the default. 
change 
`align-items: center; ` to 
`align-items: stretch; `
should solve the problem

Reply:
ok! totally missed this.